lib/portal/dependencies.properties

```diff
@@ -6,16 +6,19 @@ hibernate-core=org.hibernate:hibernate-core:5.6.7.Final
 hikaricp=com.zaxxer:HikariCP:2.6.3
 istack-commons-runtime=com.sun.istack:istack-commons-runtime:3.0.7
 jabsorb=com.liferay:org.jabsorb:1.3.2.LIFERAY-PATCHED-1
+jackson-databind=com.fasterxml.jackson.core:jackson-databind:2.13.2.2
```

Comment:
@mbowerman We are trying to move libs defined here out as possible as we can now, the goal is to remove this file in the end. So adding new entries are not acceptable now. Could you please see if you have any other solution to avoid doing this? If you have any difficulties, please feel free to come to me, we can have a talk.

Thanks.
Tina.

Reply:
@javalosjr96 Could you manually remove the changes that were made to this class? I think it probably got changed automatically when you ran the source formatter, due to some old `source-formatter.properties` changes that had never been applied to this class.

Reply:
@mbowerman yessir